### PR TITLE
Fix db-init script for branch rhbq-2.7

### PR DESCRIPTION
**Pull request by juliaaano**

### CODE REVIEW

File: rest-heroes/src/main/kubernetes/minikube.yml [hunk 1]
```diff
@@ -51,7 +51,7 @@ data:
   get-data.sh: |-
     #!/bin/bash
     
-    curl https://raw.githubusercontent.com/quarkusio/quarkus-super-heroes/main/rest-heroes/deploy/db-init/initialize-tables.sql --output /docker-entrypoint-initdb.d/1-init-tables.sql
+    curl https://raw.githubusercontent.com/quarkusio/quarkus-super-heroes/rhbq-2.7rest-heroes/deploy/db-init/initialize-tables.sql --output /docker-entrypoint-initdb.d/1-init-tables.sql
```
Comment (edeandrea) [inline rest-heroes/src/main/kubernetes/minikube.yml:54]: I think you missed a `/` between `rhbq-2.7` and `rest-heroes`

**Review by edeandrea — CHANGES_REQUESTED** [review]
Thank you very much for this @juliaaano ! I can't believe we missed this!

I think there is 1 typo that I commented on in `rest-heroes/src/main/kubernetes/minikube.yml`.

File: rest-heroes/src/main/kubernetes/minikube.yml (hunk 1 above)
Comment (juliaaano) [reply rest-heroes/src/main/kubernetes/minikube.yml]: What a nice catch. That's what happens when you get overly excited while making your first contribution to a project. Thank you.

**Review by edeandrea — APPROVED** [review]
LGTM! Thanks again!

**Pull request merged**